docs/tutorials/steps_to_train_models_with_multi_gpus_using_dp.md

```diff
@@ -0,0 +1,57 @@
+1) If you wish to conduct the training process on specific GPUs, you can set it by providing the device IDs separated by spaces:
+
+- To use 1 GPU (with ID 0):
+```
+python ./scripts/run_benchmark.py --config-path "rolling_forecast_config.json" --data-name-list "Electricity.csv" --strategy-args '{"horizon":96}' --model-name "time_series_library.PatchTST" --model-hyper-params '{"batch_size": 16, "d_ff": 1024, "d_model": 512, "lr": 0.001, "horizon": 96, "seq_len": 512}' --adapter "transformer_adapter"  `--gpus 0`  --num-workers 1  --timeout 60000  --save-path "Electricity/PatchTST"
+```
+- To use 2 GPUs (with IDs 0 and 2):
+```
+python ./scripts/run_benchmark.py --config-path "rolling_forecast_config.json" --data-name-list "Electricity.csv" --strategy-args '{"horizon":96}' --model-name "time_series_library.PatchTST" --model-hyper-params '{"batch_size": 16, "d_ff": 1024, "d_model": 512, "lr": 0.001, "horizon": 96, "seq_len": 512}' --adapter "transformer_adapter"  `--gpus 0 2`  --num-workers 1  --timeout 60000  --save-path "Electricity/PatchTST"
+```
+- To use 3 GPUs (with IDs 1, 2, and 3):
+```
+python ./scripts/run_benchmark.py --config-path "rolling_forecast_config.json" --data-name-list "Electricity.csv" --strategy-args '{"horizon":96}' --model-name "time_series_library.PatchTST" --model-hyper-params '{"batch_size": 16, "d_ff": 1024, "d_model": 512, "lr": 0.001, "horizon": 96, "seq_len": 512}' --adapter "transformer_adapter"  `--gpus 1 2 3`  --num-workers 1  --timeout 60000  --save-path "Electricity/PatchTST"
+```
+2) To use all the GPUs that have been detected, you can set the `gpus` argument to -1:
+
+- To use all GPUs (with IDs 0, 1, 2, and 3):
+```
+python ./scripts/run_benchmark.py --config-path "rolling_forecast_config.json" --data-name-list "Electricity.csv" --strategy-args '{"horizon":96}' --model-name "time_series_library.PatchTST" --model-hyper-params '{"batch_size": 16, "d_ff": 1024, "d_model": 512, "lr": 0.001, "horizon": 96, "seq_len": 512}' --adapter "transformer_adapter"  `--gpus -1`  --num-workers 1  --timeout 60000  --save-path "Electricity/PatchTST"
+```
+
+### (Optional) 3. Configure the `use_multi_gpu` Argument within the `model-hyper-params` Argument
+
+It is important to note that, by default, the `use_multi_gpu` parameter is set to `True` even without explicit specification in the launching script. However, in the event that you intend to train the models on a single GPU within a multi-GPU machine, you have the option to set it as `False`.
+
+python ./scripts/run_benchmark.py --config-path "rolling_forecast_config.json" --data-name-list "Electricity.csv" --strategy-args '{"horizon":96}' --model-name "time_series_library.PatchTST" --model-hyper-params '{"batch_size": 16, "d_ff": 1024, "d_model": 512, "lr": 0.001, "horizon": 96, "seq_len": 512, `"use_multi_gpu": "True"`}' --adapter "transformer_adapter"  --gpus -1  --num-workers 1  --timeout 60000  --save-path "Electricity/PatchTST"
```

Comment:
1. I suggest to use a more general parameter for how to use multiple gpus. Rather than 'use_multi_gpu', I think a string-typed 'parallel_strategy' should be better, and the only supported values by now are None and 'DP'/'dp', please add checks for this in the tslib adapter.
2. Using a codeblock here can make a cleaner visual effect, though it becomes more difficult to highlight the change in the command, but you can still do something like this:
```markdown
python ./scripts/run_benchmark.py --config-path "rolling_forecast_config.json" \
  --data-name-list "Electricity.csv" --strategy-args '{"horizon":96}' --model-name "time_series_library.PatchTST" \
  --model-hyper-params '{"use_multi_gpu": true, "batch_size": 16, "d_ff": 1024, "d_model": 512, "lr": 0.001, "horizon": 96, "seq_len": 512}' \
  # NEW PARAM HERE ->     ^^^^^^^^^^^^^^^^^^^^
  --adapter "transformer_adapter"  --gpus -1  --num-workers 1  --timeout 60000  --save-path "Electricity/PatchTST"
```
@qiu69 please decide how present the changed command in the documents, given the overall style in the documentation should be consistent.